request_a_govuk_domain/request/admin/model_admins.py

```diff
@@ -22,10 +23,158 @@ def has_module_permission(self, request):
         return False
 
 
-class ReviewInline(admin.StackedInline):
+class ReviewAdmin(admin.ModelAdmin):
     model = Review
-    can_delete = False
-    verbose_name_plural = "Reviews"
+    form = ReviewForm
+
+    list_display = (
+        "get_reference",
+        "get_domain_name",
+        "get_status",
+        "get_registrar_org",
+        "get_registrant_org",
+        "get_time_submitted",
+        "get_owner",
+    )
+
+    def _get_formatted_display_fields(self, display_fields: dict) -> str:
+        formatted_fields = ""
+        for key, value in display_fields.items():
+            formatted_fields = f"{formatted_fields}<li>{key} : {value}</li>"
```

Comment:
could this somehow use HTML templates? Might be hard to find if the markup needs to be changed (to add a class to `<li>` for instance)